src/wolframite/base/package.clj

```diff
@@ -0,0 +1,49 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as string]
+   [wolframite.core :as wl]
+   [wolframite.impl.wolfram-syms.intern :as wi]
+   [wolframite.wolfram :as w]))
+
+(defn load-symbols
+  "Finds all of the public symbols in the given context and interns them to a clojure-accesible namespace. This is put under the given symbol or the context by default.
```

Comment:
is "context" the same as "package"? If not, what is the difference/similarity between them?

Reply:
They're not exactly the same. I should add some notes to the glossary though. Here's an overview ("https://mathematica.stackexchange.com/questions/29324/creating-mathematica-packages"). In practice, a context is like a namespace, whereas a package is a file that defines/contains a namespace and the interface to said namespace.